File: courses/C# 2008 - Impacta/C# 2008 - M2/CS25102009/ExcelReferencia13/AcessaExcel/AcessaExcel/Form1.vb
Public Class Form1

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        Dim xl As New Excel.Application

        With xl
            .Visible = True

            .Workbooks.Add()

            .Cells(3, 4) = TextBox1.Text


            .ActiveWorkbook.Close(False)
        End With

        xl = Nothing
    End Sub
End Class

File: courses/C# 2008 - Impacta/C# 2008 - M2/CS25102009/ExcelComCSharpII/ExcelComCSharpII/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace ExcelComCSharpII
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            Excel.Application xl = new Excel.Application();

            xl.Visible = true;
            xl.Workbooks.Add("c:\\Pasta1.xlt");

            xl.Cells[3, 4] = TextBox1.Text;

        }
    }
}
